examples/liqpool/src/lib.rs

```diff
@@ -340,35 +340,35 @@ mod test {
         let og_host = swap_mock_host(host);
 
         let addr_p = AccountId(PublicKey::PublicKeyTypeEd25519(Uint256(
-            PublicKeyEd25519::from_string(
+            StrkeyPublicKey::from_string(
```

Comment:
@tomerweller The Strkey and XDR type will often be used together. I had suggested we drop the "Ed25519" from the strkey type, but maybe we should have kept the Ed25519 as it would parallel the XDR type. Wdyt?